### ISSUE TITLE
API Gateway Ingress Controller without IAM User

### ISSUE THREAD

**Issue by praveen5959** [aws-samples/amazon-apigateway-ingress-controller-blog#2]
I am trying to deploy the API Gateway Ingress Controller without IAM User. Due to some security reasons I cannot create the IAM User. Is there any way to deploy the controller without the IAM User and say using a IAM role or anything?

**Comment by amitoo7** [comment]
Yes, You can API Key in API Gateway in this case you don't need to create IAM user if you don't want to use it.
